src/router/index.tsx

```
@@ -0,0 +1,88 @@
+import { createBrowserRouter } from 'react-router-dom';
+
+import { ROUTES } from '@/constants/paths/routes';
+import { Error } from '@/pages';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)